tensorflow/contrib/memory_stats/python/kernel_tests/memory_stats_ops_test.py

```diff
@@ -65,8 +65,11 @@ def testChainOfMatmul(self):
       sess.run(d)
 
       max_bytes_in_use = sess.run(memory_stats_ops.MaxBytesInUse())
+      bytes_in_use = sess.run(memory_stats_ops.BytesInUse())
       self.assertGreaterEqual(max_bytes_in_use, matrix_size_in_bytes * 3)
       self.assertLess(max_bytes_in_use, matrix_size_in_bytes * 4)
+      self.assertGreaterEqual(bytes_in_use, matrix_size_in_bytes * 3)
```

Comment:
The new tests don't distinguish BytesInUse from MaxBytesInUse. One idea is to launch a second session with a smaller graph. MaxBytesInUse should return the larger size, and BytesInUse should return the smaller size.